//= error code 2

fun main() {
    exit(2);
}